LayoutKit/Views/ReloadableView.swift

```diff
@@ -80,12 +57,28 @@ extension UICollectionView: ReloadableView {
         registerClass(UICollectionReusableView.self, forSupplementaryViewOfKind: UICollectionElementKindSectionFooter, withReuseIdentifier: reuseIdentifier)
     }
 
-    public func insert(indexPaths indexPaths: [NSIndexPath]) {
-        insertItemsAtIndexPaths(indexPaths)
-    }
+    public func perform(batchUpdates batchUpdates: BatchUpdates) {
```

Comment:
If this is public, do we want to add docs for this? Also, we should document somewhere the order of these operations because it matters